src/shared/locales/ptBr/translations/patient/index.ts

```diff
@@ -84,14 +112,116 @@ export default {
       },
       addNoteAbove: 'Adicione uma nota utilizando o botão acima.',
     },
+    labs: {
+      label: 'Laboratórios',
+      new: 'Adicionar novo laboratório',
+      warning: {
+        noLabs: 'Sem laboratórios',
+      },
+      noLabsMessage: 'Sem solicitações de laboatórios para essa pessoa.',
+    },    
     medications: {
       label: 'Remédios',
       new: 'Adicionar novo medicamento',
       warning: {
-        noMedications: 'Nomejikachion s',
+        noMedications: 'Sem medicamentos',
       },
       noMedicationsMessage: 'Nenhum pedido de medicamentos para esta pessoa.',
     },
+    careGoal: {
+      new: 'Adicionar meta de cuidado',
+      label: 'Meta de cuidado',
+      title: 'Título',
+      description: 'Descrição',
+      status: 'Status',
+      achievementStatus: 'Status da conquista',
+      priority: {
+        label: 'Prioridade',
+        low: 'baixa',
+        medium: 'média',
+        high: 'alta',
+      },
+      startDate: 'Data de ínicio',
+      dueDate: 'Data de vencimento',
+      note: 'Nota',
+      error: {
+        unableToAdd: 'Incapaz de adicionar uma nova meta de cuidado. ',
+        descriptionRequired: 'A descrição é obrigatória. ',
+        priorityRequired: 'Priority is required.',
+        statusRequired: 'A prioridade é obrigatória. ',
+        achievementStatusRequired: 'O status de conquista é obrigatório. ',
+        startDate: 'A data de início é obrigatória. ',
+        dueDate: 'A data de vencimento é obrigatória. ',
+        dueDateMustBeAfterStartDate: 'A data de vencimento deve ser posterior à data de início ',
+      },
+    },
+    careGoals: {
+      warning: {
+        noCareGoals: 'Sem metas de cuidado.',
+        addCareGoalAbove: 'Adicione uma meta de cuidado usando o botão acima. ',
+      },
+    },
+    carePlan: {
+      new: 'Adicionar plano de saúde',
+      label: 'Planos de saúde',
+      title: 'Título',
+      description: 'Descrição',
+      status: 'Status',
+      condition: 'Condição',
+      intent: 'Intent',
```

Comment:
English

Reply:
This Intent I left in English because I don't know for sure if the literal translation is the right translation.

Reply:
Thanks @tsdamas.
I'll change it